ext/drbbdup/drbbdup.c

```diff
@@ -712,16 +714,20 @@ drbbdup_encode_runtime_case(void *drcontext, drbbdup_per_thread *pt, void *tag,
      * dispatcher.
      */
     drreg_are_aflags_dead(drcontext, where, &manager->are_flags_dead);
-    if (!manager->is_scratch_reg_dead)
```

Comment:
So this was a mistake before causing the scratch to *always* be spilled, right?  There wasn't some subtle reason this was checked before drreg_is_register_dead was called?

Reply:
Nice catch! No reason.